docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/refget/refget.md

```diff
@@ -0,0 +1,246 @@
+
+## Things to be aware of when using refget
+
+### Coordinates used in refget
+
+The refget protocol uses the `0-start, half-open` coordinate system, which is found in other formats such as BED, SAM/BAM/CRAM and utilised by the UCSC Genome Browser. This system is sometimes called 0-based. In this system you start counting sequence positions from 0. More information about this system is available from the [UCSC Genome Browser team's blog site](https://genome-blog.gi.ucsc.edu/blog/2016/12/12/the-ucsc-genome-browser-coordinate-counting-systems/).
+
+When converting from Ensembl coordinates (1-start, fully-closed; sometimes called 1-based) you need only remove `1` from the start. The end remains the same. For example, given an Ensembl coordinate (1-based) 1:12714000**1**-127140001, this is represented as 1:12714000**0**-127140001 (0-based).
+
+### Using appropriate `Accept` content-types
+
+Refget is particular about the content-types it is provided. When requesting a sequence you must specify `plain/text` as the content type and when accessing sequence metadata you must specify `application/json`. Some URLs will fail when clicked on within a web browser due to their automated provision of Accept content types.
+
+### Why a metadata mentions multiple aliases for Ensembl sequences rather than just one
+
+The refget system effectively creates a non-redundant database of sequences. If the same sequence of characters appear in the same order they will result in the same checksum identifier being generated irrespective of molecule type (nucleotides versus amino acid residues).
+
+### Use of multiple identifiers for a sequence
+
+Refget's specification details two checksum algorithms which can be used to generate the identifier. These are a hex encoded `MD5` or a scheme called [`sha512t24u`](https://samtools.github.io/hts-specs/refget.html#refget-checksum-algorithm). A GA4GH identifier is generated by adding `ga4gh:SQ.` to a `sh512t24u` checksum e.g. `ga4gh:SQ.0qw_sn8Cl7OmMTFlukjFD2DUejW0T80Y` and `17dab79b963ccd8e7377cef59a54fe1c` represent the same sequence.
+
+## Third party refget clients
+
+You can use the [Python `refget`](https://pypi.org/project/refget/) library to fetch sequences from the Ensembl refget instance.
+
+## Accessing sequence data for features in Ensembl
+
+The ["Getting started with Graphql"](../tools-and-apis/graphql/using-graphql.md) and ["Using Graphql"](../tools-and-apis/graphql/using-graphql.md) pages cover how to access data through Ensembl GraphQL service and the example below assumes that this service is being used.
```

Comment:
> The ["Getting started with Graphql"](../tools-and-apis/graphql/using-graphql.md)

I am afraid this is not how links in the help app work right now. When we designed the help app, it was suggested that all generated urls for articles were top-level in the `articles` directory, because of 2.5 reasons: 1) we didn't have index pages for directiories (e.g. there would be no page corresponding to `../tools-and-apis` or `../tools-and-apis/graphql`); 2) Lucy favoured url readability; 3) _(half a reason)_ I was saying that deeply nested urls were worse for search engines (a common opinion among seo folks; but could be rubbish, because it is all black magic).

As a result, the url for the using-graphql article is https://beta.ensembl.org/help/articles/using-graph-ql. I do not know why the slugify module that created a url slug out of the file name insists on putting a hyphen before the ql.